ui/raidboss/data/03-hw/alliance/weeping_city.txt

```diff
@@ -0,0 +1,418 @@
+1060.7 "Megiddo Flame" sync /:Forgall:17C0:/
+1066.7 "Punishing Ray" sync /:Forgall:17C4:/
+1081.0 "Brand of the Fallen" sync /:Forgall:17CC:/ window 20,20
+1090.3 "Evil Mist" sync /:Forgall:17C5:/
+1101.3 "Necropurge" sync /:Shriveled Talon:17D7:/
+1118.3 "Necropurge" sync /:Forgall:17BE:/
+1131.6 "Necropurge" sync /:Forgall:17BF:/
+1132.8 "Brand of the Fallen" sync /:Forgall:17CC:/ window 20,20
+1141.1 "--sync--" sync /:Forgall:17C2:/ # Dark Eruption markers go out.
+1146.3 "Dark Eruption x3" # sync /:Forgall:17C3:/
+1163.2 "--sync--" sync /:Forgall:17C7:/ window 163.2 # Pushes at 50% HP.
+
+# We don't know how long the intermission goes.
+# There's no data available to tell us what, if anything,
+# happens if the three adds aren't killed in time.
+# It could even be nothing.
+# If the intermission goes longer than 3 minutes, something is WRONG.
+
+1350.0 "Mana Explosion" sync /:Forgall:17C8:/ window 350,5
+
+1360.2 "Necropurge" sync /:Forgall:17BE:/ window 200,5
+1373.4 "Necropurge" sync /:Forgall:17BF:/
+1373.4 "--sync--" sync /:Poison Mist:17D8:/
+1377.5 "Mega Death" sync /:Forgall:17CA:/ window 20,20
+1388.5 "--sync--" sync /:Forgall:17C2:/
+1393.7 "Dark Eruption x3" # sync /:Forgall:17C3:/
+1395.7 "Megiddo Flame" sync /:Forgall:17C0:/
+1400.9 "Evil Mist" sync /:Forgall:17C5:/
+1414.9 "Necropurge" sync /:Forgall:17BF:/
+1414.9 "--sync--" sync /:Poison Mist:17D8:/
+1419.1 "Mega Death" sync /:Forgall:17CA:/ window 20,20
+1439.2 "Hell Wind" sync /:Forgall:17CB:/
+1453.3 "Brand of the Fallen" sync /:Forgall:17CC:/
+1459.5 "Punishing Ray" sync /:Forgall:17C4:/
+1465.6 "Brand of the Fallen" sync /:Forgall:17CC:/
+
+1481.9 "Necropurge" sync /:Forgall:17BE:/ jump 1360.2
+1495.1 "Necropurge"
+1499.1 "Mega Death"
+1515.4 "Dark Eruption x3"
+
+#~~~~~~#
+# OZMA #
+#~~~~~~#
+
+# -ii 366 367 368 1802 1806 1807 1829 1830 1832
+# -ic "Singularity Ripple" "Singularity Trace"
+
+# Ozma is based on an A/B timeline pattern.
+# She begins in Sphere form and can morph into Cube or Pyramid stance.
+# The complete flow looks like:
+
+# Sphere opening block -->              --> Cube bridge block ---->       . - - - - - - - - - - ---
+#                                                                         V                       |
+# Cube opening block ----> Intermission --> Sphere bridge block --> (Cube|Sphere) rotation blocks |
+
+2000.0 "--sync--" sync /00:0839:The Gloriole will be sealed off/ window 2000,5
+2015.0 "Meteor Headmarkers"
+2024.0 "Meteor Impact" sync /:Singularity Fragment:1935:/
+
+2045.5 "Transfiguration (Pyramid/Cube?)" sync /:Ozma:1826:/ jump 2150.0
+2045.5 "--sync--" sync /:Ozma:1803:/ jump 2250.0
+2052.6 "Execration?"
+2052.6 "Flare Star?"
+2059.0 "Explosion x5?"
+2073.9 "Acceleration Bomb?"
+
+# Pyramid-first path
+
+2150.0 "Transfiguration (Pyramid)" sync /:Ozma:1826:/
+2157.1 "Execration" sync /:Ozma:1828:/
+2178.4 "Acceleration Bomb" sync /:Ozma:182F:/
+2195.7 "Transfiguration (Sphere)" sync /:Ozma:1827:/
+2203.8 "Black Hole"
```

Comment:
Is this missing a jump like 2308 is?

Also, could you add a --sync-- on the starts casting because this is an HP push?

Reply:
I removed the sync from this one because it was doing weird stuff, although I can't remember what that weird stuff was now. I figured that having a wide sync on the second incarnation would cover things. Is that a best-practices thing?

Reply:
I would say ideally yeah it's better to sync as early as possible.  I don't know that I follow that in all cases, but I think it's good, too?